### ISSUE TITLE
improve error messages on git azure init

### ISSUE THREAD

**Issue by woloski** [tjanczuk/git-azure#36]
When running this:

`git azure init --publishSettings ... --servieName .. --username .. --password ..`

I get

```
undefined
  error: unknown option `%s'
undefined
```

**Comment by tjanczuk** [comment]
Yes, there is a bunch of places like this, I need to go clean it up. Thanks for filing. 

---

From: Matias Woloski [reply@reply.github.com]
Sent: Wednesday, May 23, 2012 6:24 PM
To: Tomasz Janczuk
Subject: [git-azure] improve error messages on git azure init (#36)

When running this:

`git azure init --publishSettings ... --servieName .. --username .. --password ..`

I get

```
undefined
  error: unknown option `%s'
undefined
```

---

Reply to this email directly or view it on GitHub:
https://github.com/tjanczuk/git-azure/issues/36
